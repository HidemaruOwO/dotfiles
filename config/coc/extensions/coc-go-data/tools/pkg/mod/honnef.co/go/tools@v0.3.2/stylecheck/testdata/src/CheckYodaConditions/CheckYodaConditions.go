// Package pkg ...
package pkg

func fn(x string, y int) {
	if "" == x { //@ diag(`Yoda`)
	}
	if 0 == y { //@ diag(`Yoda`)
	}
	if 0 > y {
	}
	if "" == "" {
	}

	if "" == "" || 0 == y { //@ diag(`Yoda`)
	}
}
