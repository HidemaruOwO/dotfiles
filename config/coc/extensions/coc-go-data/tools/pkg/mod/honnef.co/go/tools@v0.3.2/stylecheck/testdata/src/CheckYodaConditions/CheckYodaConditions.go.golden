// Package pkg ...
package pkg

func fn(x string, y int) {
	if x == "" { //@ diag(`Yoda`)
	}
	if y == 0 { //@ diag(`Yoda`)
	}
	if 0 > y {
	}
	if "" == "" {
	}

	if "" == "" || y == 0 { //@ diag(`Yoda`)
	}
}
